integrations.json

```diff
@@ -75,6 +75,25 @@
       }
     ]
   },
+  {
+    "name": "aspNetWeb5",
```

Comment:
It's officially called ASP.NET Web API 2 (even though the NuGet packages are versioned 5.x), so let's call this `aspNetWebApi2`.